editor/src/components/editor/store/editor-update.tsx

```diff
@@ -320,8 +320,8 @@ export function runSimpleLocalEditorAction(
       return UPDATE_FNS.SET_FILEBROWSER_DROPTARGET(action, state)
     case 'SET_FORKED_FROM_PROJECT_ID':
       return UPDATE_FNS.SET_FORKED_FROM_PROJECT_ID(action, state)
-    case 'SET_CURRENT_THEME':
-      return UPDATE_FNS.SET_CURRENT_THEME(action, state)
+    // case 'SET_CURRENT_THEME':
```

Comment:
delete me

Reply:
No no, use me instead of calling `UPDATE_FNS.SET_CURRENT_THEME` inside `dispatch.tsx`

Reply:
I moved it from here because functions here expect a return type of `EditorState`, but this function modifies the `UserState`